src/config.js

```diff
@@ -51,6 +51,17 @@ const getAuth0Hostname = () => {
   }
 };
 
+const listeDomain = () => {
+  switch (process.env.NDLA_ENVIRONMENT) {
+    case 'prod':
+      return 'https://liste.ndla.no';
+    case 'staging':
+      return 'https://liste.staging.ndla.no';
+    default:
+      return 'https://liste.test.ndla.no';
```

Comment:
Dette burde fungere.
```suggestion
    case 'local':
      return 'http://localhost:30020';
    case 'prod':
      return 'https://liste.ndla.no';
    default:
      return `https://liste.${process.env.NDLA_ENVIRONMENT}.ndla.no`;
```